tests/runtest.py

```diff
@@ -852,6 +852,8 @@ def parse_argument():
     res.append(TestBase.TEST_UNSUPP_LANG)
     res.append(TestBase.TEST_SKIP)
 
+    if os.path.isfile("./failed-tests.txt"):
+        os.rename("./failed-tests.txt", "./failed-tests.txt.old")
```

Comment:
How about this?
```suggestion
    if os.path.exists("failed-tests.txt"):
        os.rename("failed-tests.txt", "failed-tests.txt.old")
```

Reply:
I think that your suggestion is more appropriate. I'll update it. Thanks!!!